### PR TITLE
LPD-44303 - Final removal of @liferay/npm-scripts

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by izaera** [comment]
> Final removal of @liferay/npm-scripts

astounding

**Review by izaera — APPROVED** [review]
If it doesn't break anything in CI or build, LWTM :tada:

File: modules/_node-scripts/bundle/custom.mjs [hunk 1]
```diff
@@ -6,14 +6,39 @@
...
+
+		console.log(`Bundle served at 'localhost:${port}'`);
+	}
+	else {
+		await Promise.all([
+			customBuildConfig.other
+				? Promise.resolve(customBuildConfig.other())
+				: Promise.resolve(),
+			esbuild.build({
+				minify: process.env.NODE_ENV === 'production',
```
Comment (dsanz) [inline modules/_node-scripts/bundle/custom.mjs:39]: didn't we have this before?

**Comment by bryceosterhaus** [comment]
ci:report:96478874

File: modules/_node-scripts/bundle/custom.mjs (hunk 1 above)
Comment (bryceosterhaus) [reply modules/_node-scripts/bundle/custom.mjs:39]: We didn't have it on by default for "build:custom" we do have it on for normal builds

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:report:98680255

**Comment by bryceosterhaus** [comment]
ci:test:stable

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:report:98869340

**Comment by bryceosterhaus** [comment]
ci:test:stable

**Comment by bryceosterhaus** [comment]
ci:report:100115556

**Comment by bryceosterhaus** [comment]
ci:test:stable

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:report:100287896

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:report:100367646

**Comment by bryceosterhaus** [comment]
ci:report:100367646

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:report:101572084

**Comment by bryceosterhaus** [comment]
ci:forward:force

**Pull request closed without merging**